src/lsm/tree.zig

```diff
@@ -167,6 +167,10 @@ pub fn TreeType(comptime TreeTable: type, comptime Storage: type, comptime tree_
         filter_block_hits: u64 = 0,
         filter_block_misses: u64 = 0,
 
+        // [key_min, key_max] is the range of keys in this tree.
+        key_min: ?Key,
+        key_max: ?Key,
```

Comment:
This should probably be an optional KeyRange type, since I believe that both are null or present together as one.